### PR TITLE
fix(amplify-category-auth): fixed no parameter when hostedui is not p…

### PR DESCRIPTION
**Pull request by ammarkarachi**
…resent

#### Description of changes
Set hostedUI to false when federation is removed in update flow

#### Description of how you validated changes
yarn e2e src/__tests__/auth*

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by johnpc — APPROVED** [review]
lgtm

File: packages/amplify-e2e-core/src/categories/auth.ts [hunk 1]
```diff
@@ -368,6 +368,23 @@ export function updateAuthSignInSignOutUrl(cwd: string, settings: any): Promise<
   });
 }
 
+export function updateAuthToRemoveFederation(cwd: string, settings: any): Promise<void> {
```
Comment (jhockett) [inline packages/amplify-e2e-core/src/categories/auth.ts:371]: nit: exclude settings parameter if not used.

**Review by jhockett — APPROVED** [review]
1 minor nit, but LGTM

File: packages/amplify-e2e-core/src/categories/auth.ts (hunk 1 above)
Comment (ammarkarachi) [reply packages/amplify-e2e-core/src/categories/auth.ts:371]: I will get this in a follow up PR

**Pull request merged**